webscalesqlclient/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ cmake_minimum_required(VERSION 2.8.0)
 include(ExternalProject)
 ExternalProject_Add(
   webscalesqlclient
+  URL file://${CMAKE_CURRENT_SOURCE_DIR}/webscalesql-5.6
```

Comment:
Something in the cmake process appears to be removing the contents of that directory on a completely clean build.